internal/accrual/accrual.go

```diff
@@ -0,0 +1,36 @@
+package accrual
+
+import (
+	"encoding/json"
+	"github.com/go-resty/resty/v2"
+)
+
+type RegisterResponse struct {
+	Order   string  `json:"order"`
+	Status  string  `json:"status"`
+	Accrual float32 `json:"accrual"`
+}
+
+func GetOrderInfo(accrualServerAddress string, orderNumber string) (RegisterResponse, error) {
+	var registerResponse RegisterResponse
+	req := resty.New().
+		SetBaseURL(accrualServerAddress).
+		R().
+		SetHeader("Content-Type", "application/json")
+
+	resp, err := req.Get("/api/orders/" + orderNumber)
+
+	if err != nil {
+		return registerResponse, err
+	}
+
+	if resp.StatusCode() != 200 {
+		return registerResponse, err
```

Comment:
Хорошая практика "оборачивать" ошибки, добавляя к ним дополнительный контекст. Это поможет разобраться с тем, в каком месте приложения произошел сбой и с чем он связан, когда ошибка будет залогирована.
```suggestion
		return registerResponse, fmt.Errorf("unexpected status code: %d", resp.StatusCode())
```